Tracker/Helpers/Extensions/Date+Extensions.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Date+Extensions.swift
+//  Tracker
+//
+//  Created by Andrei Kashin on 08.05.2023.
+//
+
+import Foundation
+
+extension Date {
+    static func from(year: Int, month: Int, day: Int) -> Date? {
+        let calendar = Calendar(identifier: .gregorian)
+        var dateComponents = DateComponents()
+        dateComponents.year = year
+        dateComponents.month = month
+        dateComponents.day = day
+        return calendar.date(from: dateComponents) ?? nil
```

Comment:
опциональный метод в случае чего сам по себе вернет nil
```suggestion
        return calendar.date(from: dateComponents)
```